src/Core/Geographic/Extent.js

```diff
@@ -244,15 +251,15 @@ class Extent {
         const c = (this.crs == coord.crs) ? coord : coord.as(this.crs, _c);
         // TODO this ignores altitude
         if (CRS.isGeographic(this.crs)) {
-            return c.longitude() <= this.east + epsilon &&
-                   c.longitude() >= this.west - epsilon &&
-                   c.latitude() <= this.north + epsilon &&
-                   c.latitude() >= this.south - epsilon;
+            return c.longitude <= this.east + epsilon &&
```

Comment:
you could return directly  the second part (the `else` part)  with x and z.

Reply:
Yes, didn't see this at all...